torch/ao/quantization/qconfig.py

```diff
@@ -50,7 +62,12 @@ class QConfig(namedtuple('QConfig', ['activation', 'weight'])):
     Note that QConfig needs to contain observer **classes** (like MinMaxObserver) or a callable that returns
     instances on invocation, not the concrete observer instances themselves.
     Quantization preparation function will instantiate observers multiple times for each of the layers.
-
+    All fields are optional but there are some constraints:
+    * when `bias` observer is not provided, we'll infer the dtype of bias from input
+    activation and weight
+    * when `input_activation` or `output_activation` configs are None, we'll use
```

Comment:
would it be possible to remove this and just have `activation` which means output activation (same as before), and add `input_activation`?  It would be good to make the config as simple as possible.